### DISCUSSION TITLE
Want to give some feedback on GitHub Packages?

### DISCUSSION THREAD

**Discussion by jnstuart** [community/community#45938] [Actions]
### Select Topic Area

Product Feedback

### Body

The GitHub Packages team is looking to talk to **developers who publish packages to GitHub Package Registry, NPMjs.org, or NuGet.org**. If you're interested in chatting with us for 30 min about how you use these package registries, please complete this survey. We'd be happy to provide a $50 gift card as a thank you for your time.

[https://github.surveymonkey.com/r/9FRD8VT](https://github.surveymonkey.com/r/9FRD8VT)

**Comment by noirbizarre** [comment]
Just so you know:
I filled the form, but the 4th field (Which of the following GitHub products you regularly use? aka. multichoices) is broken

**Comment by ghost** [comment]
Is the GitHub Packages team as useless as the GitHub Support team?

**Comment by VNRARA** [comment]
I can't even use GitHub Packages with Docker Gui via Synology. Useless.
